mods/content/sexcon/_sexcon.dm

```diff
@@ -0,0 +1,84 @@
+/decl/modpack/sexcon
+	name = "Sex Controller Content"
+
+/decl/config/toggle/sexcon
+	uid = "toggle_sexcon"
+	desc = "Remove the # to enable sex mechanics."
+
+/decl/config/toggle/sexcon/Initialize()
+	// Init decls repository for sex actions.
+	decls_repository.get_decls_of_subtype(/decl/sex_action)
+	return ..()
+
+/datum/client_preference/sexable
+	description = "Sex Mechanics"
+	key = "SEXCON_TOGGLE"
+	options = list(PREF_NO, PREF_YES)
```

Comment:
```suggestion
	default_value = PREF_NO
```
you could also just use the default order but have it default to `PREF_NO`. these are identical except one swaps the display order.